escape-room/doc/research.bib

```diff
@@ -0,0 +1,10 @@
+@article{su_2024,
```

Comment:
ist diese bibtex-definition (und der name in einem für euch brauchbaren format?

Reply:
ich bevorzuge etwas "sprechendere" keys, die man bei der verwendung im text leichter entschlüsseln kann.